package.json

```diff
@@ -19,33 +19,26 @@
   "license": "ISC",
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/blackbaud/skyux-sample.git"
+    "url": "git+https://github.com/blackbaud/skyux-phone-field.git"
   },
   "bugs": {
-    "url": "https://github.com/blackbaud/skyux-sample/issues"
+    "url": "https://github.com/blackbaud/skyux-phone-field/issues"
+  },
+  "homepage": "https://github.com/blackbaud/skyux-phone-field#readme",
+  "peerDependencies": {
+    "@angular/common": ">=4.3.6",
+    "@angular/core": ">=4.3.6",
+    "@angular/forms": ">=4.3.6",
+    "@skyux/i18n": "^3.5.0",
+    "@skyux/theme": "^3.4.0",
+    "@skyux/popovers": "^3.0.0"
+  },
+  "dependencies": {
+    "intl-tel-input": "14.0.8"
   },
-  "homepage": "https://github.com/blackbaud/skyux-sample#readme",
-  "peerDependencies": {},
-  "dependencies": {},
   "devDependencies": {
-    "@angular/animations": "4.3.6",
-    "@angular/common": "4.3.6",
-    "@angular/compiler": "4.3.6",
-    "@angular/compiler-cli": "4.3.6",
-    "@angular/core": "4.3.6",
-    "@angular/forms": "4.3.6",
-    "@angular/http": "4.3.6",
-    "@angular/platform-browser": "4.3.6",
-    "@angular/platform-browser-dynamic": "4.3.6",
-    "@angular/router": "4.3.6",
-    "@blackbaud/skyux": "2.26.0",
-    "@blackbaud/skyux-builder": "1.24.0",
-    "@skyux-sdk/builder-plugin-skyux": "1.0.0-rc.0",
-    "core-js": "2.4.1",
-    "rxjs": "5.4.3",
-    "ts-node": "3.0.4",
-    "tslint": "5.2.0",
-    "typescript": "2.3.2",
-    "zone.js": "0.8.10"
+    "@blackbaud/skyux": "2.40.0",
+    "@blackbaud/skyux-builder": "1.31.1",
+    "@skyux-sdk/builder-plugin-skyux": "1.0.0-rc.6"
```

Comment:
While you've got the hood open, you could probably update these packages to latest.

Reply:
Done